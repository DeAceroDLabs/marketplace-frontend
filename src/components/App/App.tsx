import React from "react";
import { BrowserRouter as Router, Routes, Route } from "react-router-dom";
import Home from "components/Home";
import Search from "components/Search";

const App = () => {
  return (
    <Router>
      <Routes>
        <Route path="/" element={<Home />}></Route>
        <Route path="/search" element={<Search/>}></Route>
      </Routes>
    </Router>
  );
};

export default App;
