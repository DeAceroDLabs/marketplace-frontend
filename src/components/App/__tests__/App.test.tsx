import renderer from "react-test-renderer";
import App from "../App";

it("renders App with no issue", () => {
  const component = renderer.create(<App />);
  let tree = component.toJSON();
  expect(tree).toMatchSnapshot();
});
