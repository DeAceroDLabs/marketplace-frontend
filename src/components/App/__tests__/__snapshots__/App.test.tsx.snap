// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders App with no issue 1`] = `
Array [
  <div
    className="header clean"
  >
    <div
      className="logo-container"
      id="logo"
    >
      <button
        className="logo"
        onClick={[Function]}
      >
        <svg
          aria-hidden={true}
          className="MuiSvgIcon-root MuiSvgIcon-fontSizeMedium css-i4bv87-MuiSvgIcon-root"
          data-testid="FactoryIcon"
          focusable="false"
          viewBox="0 0 24 24"
        >
          <path
            d="M22 10v12H2V10l7-3v2l5-2v3h8zm-4.8-1.5L18 2h3l.8 6.5h-4.6zM11 18h2v-4h-2v4zm-4 0h2v-4H7v4zm10-4h-2v4h2v-4z"
          />
        </svg>
      </button>
    </div>
    <div
      className="login-container"
    >
      <button
        className="primary"
        onClick={[Function]}
      >
        <div
          className="person-icon"
          id="user-icon"
        >
          <svg
            aria-hidden={true}
            className="MuiSvgIcon-root MuiSvgIcon-fontSizeMedium css-12jf5c-MuiSvgIcon-root"
            data-testid="PersonIcon"
            focusable="false"
            viewBox="0 0 24 24"
          >
            <path
              d="M12 12c2.21 0 4-1.79 4-4s-1.79-4-4-4-4 1.79-4 4 1.79 4 4 4zm0 2c-2.67 0-8 1.34-8 4v2h16v-2c0-2.66-5.33-4-8-4z"
            />
          </svg>
        </div>
        <div>
          Hola, 
          <b>
            Ingresa
          </b>
        </div>
      </button>
    </div>
    <div
      className="shopping-cart-icon"
    >
      <button
        className="primary"
        onClick={[Function]}
      >
        <svg
          aria-hidden={true}
          className="MuiSvgIcon-root MuiSvgIcon-fontSizeMedium css-18e385h-MuiSvgIcon-root"
          data-testid="ShoppingCartIcon"
          focusable="false"
          viewBox="0 0 24 24"
        >
          <path
            d="M7 18c-1.1 0-1.99.9-1.99 2S5.9 22 7 22s2-.9 2-2-.9-2-2-2zM1 2v2h2l3.6 7.59-1.35 2.45c-.16.28-.25.61-.25.96 0 1.1.9 2 2 2h12v-2H7.42c-.14 0-.25-.11-.25-.25l.03-.12.9-1.63h7.45c.75 0 1.41-.41 1.75-1.03l3.58-6.49c.08-.14.12-.31.12-.48 0-.55-.45-1-1-1H5.21l-.94-2H1zm16 16c-1.1 0-1.99.9-1.99 2s.89 2 1.99 2 2-.9 2-2-.9-2-2-2z"
          />
        </svg>
      </button>
    </div>
  </div>,
  <div
    className="header"
  >
    <div>
      <div
        className="container horizontal"
      >
        <img
          alt=""
          className="bgImage"
          src="/background.png"
        />
        <div
          className="content"
        >
          <div
            className="header-content"
          >
            <div
              className="header-title"
            >
              Somos tu aliado de confianza para hacer crecer tu negocio
            </div>
            <div
              className="search-container"
            >
              <div
                className="search-bar"
              >
                <input
                  onChange={[Function]}
                  onKeyDown={[Function]}
                  placeholder="¿Qué estás buscando?"
                  type="text"
                  value=""
                />
                <button
                  onClick={[Function]}
                >
                  <svg
                    aria-hidden={true}
                    className="MuiSvgIcon-root MuiSvgIcon-fontSizeMedium css-bq05nv-MuiSvgIcon-root"
                    data-testid="SearchIcon"
                    focusable="false"
                    viewBox="0 0 24 24"
                  >
                    <path
                      d="M15.5 14h-.79l-.28-.27C15.41 12.59 16 11.11 16 9.5 16 5.91 13.09 3 9.5 3S3 5.91 3 9.5 5.91 16 9.5 16c1.61 0 3.09-.59 4.23-1.57l.27.28v.79l5 4.99L20.49 19l-4.99-5zm-6 0C7.01 14 5 11.99 5 9.5S7.01 5 9.5 5 14 7.01 14 9.5 11.99 14 9.5 14z"
                    />
                  </svg>
                </button>
              </div>
            </div>
          </div>
          <div
            className="red"
          />
        </div>
      </div>
    </div>
    <div />
  </div>,
  <div
    className="body"
  >
    <div>
      <div
        className="section"
      >
        <div
          className="title"
        >

          Login

        </div>
        <div
          className="body"
        >

          <form
            onSubmit={[Function]}
          >
            <input
              onChange={[Function]}
              placeholder="username"
              type="text"
              value=""
            />
            <input
              placeholder="password"
              type="text"
            />
            <button
              type="submit"
            >
              Log In
            </button>
          </form>

        </div>
      </div>
    </div>
  </div>,
  <div
    className="footer"
  >
    <div />
  </div>,
]
`;
